src/views/dialogs/PredictionMarketIntroDialog.tsx

```diff
@@ -0,0 +1,106 @@
+import { useCallback, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { ButtonAction } from '@/constants/buttons';
+import type { DialogProps, PredictionMarketIntroDialogProps } from '@/constants/dialogs';
+import { LocalStorageKey } from '@/constants/localStorage';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useLocalStorage } from '@/hooks/useLocalStorage';
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { Button } from '@/components/Button';
+import { Checkbox } from '@/components/Checkbox';
+import { Dialog } from '@/components/Dialog';
+import { Icon, IconName } from '@/components/Icon';
+import { NewTag } from '@/components/Tag';
+
+export const PredictionMarketIntroDialog = ({
+  setIsOpen,
+}: DialogProps<PredictionMarketIntroDialogProps>) => {
+  const stringGetter = useStringGetter();
+  const [doNotShowAgain, setDoNotShowAgain] = useState(false);
+  const [, setHasSeenPredictionMarketsIntro] = useLocalStorage({
+    key: LocalStorageKey.HasSeenPredictionMarketsIntro,
+    defaultValue: false,
+  });
+
+  const onContinue = useCallback(() => {
+    setHasSeenPredictionMarketsIntro(doNotShowAgain);
+    setIsOpen(false);
+  }, [doNotShowAgain, setIsOpen, setHasSeenPredictionMarketsIntro]);
+
+  const renderPoint = ({
+    icon,
+    title,
+    description,
+  }: {
+    icon: IconName;
+    title: string;
+    description: string;
+  }) => (
+    <div key={title} tw="row gap-0.75 align-middle">
+      <$IconContainer>
```

Comment:
nit: should we any tw into this?

Reply:
I didn't add tw to the IconContainer because it would have been pretty long with the amount of css properties set. Wondering if we could make @tyleroooo 's script into an eslint/prettier formatter :P

Reply:
In this case I agree we should do all-or-nothing and doing all here is too big probably.